Tribler/Core/Upgrade/db72_to_pony.py

```diff
@@ -263,6 +262,25 @@ def convert_personal_channel(self):
 
         v.value = CONVERSION_FINISHED
 
+    @inlineCallbacks
+    def update_convert_total(self, amount, elapsed):
+        if self.notifier_callback:
+            self.notifier_callback("%i entries converted in %i seconds (%i e/s)" % (amount, int(elapsed),
+                                                                                    int(amount / elapsed)))
+            yield deferLater(reactor, 0.001, lambda: None)
+
+    @inlineCallbacks
+    def update_convert_progress(self, amount, total, elapsed):
+        if self.notifier_callback:
+            elapsed = 0.0001 if elapsed == 0.0 else elapsed
+            amount = amount or 1
+            est_speed = amount/elapsed
```

Comment:
Could you add a space before and after the `/`?

Reply:
OK.